src/sstc.adoc

```diff
@@ -75,60 +75,60 @@ If supervisor mode is implemented, its mip.STIP and mie.STIE are the
 interrupt-pending and interrupt-enable bits for supervisor-level timer
 interrupts. If the stimecmp register is not implemented, STIP is writable in
 mip, and may be written by M-mode software to deliver timer interrupts to
-S-mode. If the stimecmp (supervisor-mode timer compare) register is
+S-mode. If the `stimecmp` (supervisor-mode timer compare) register is
 implemented, STIP is read-only in mip and reflects the supervisor-level timer
 interrupt signal resulting from stimecmp. This timer interrupt signal is
-cleared by writing stimecmp with a value greater than the current time value.
+cleared by writing `stimecmp` with a value greater than the current time value.
 
-==== Supervisor Interrupt Registers (sip and sie)
+==== Supervisor Interrupt (`sip` and `sie`) Registers 
 
 This extension modifies the description of the STIP/STIE bits in these
 registers as follows:
 
-Bits sip.STIP and sie.STIE are the interrupt-pending and interrupt-enable bits
+Bits `sip.STIP` and `sie.STIE` are the interrupt-pending and interrupt-enable bits
```

Comment:
IIRC, we normally use monospaced font for the register name but not for the field name, e.g.

    `sip`.STIP and `sie`.STIE